pages/data-design/quick-start.mdx

```diff
@@ -0,0 +1,116 @@
+
+In this guide we'll walk through the process of setting up your tracking plan in Avo, suggesting your first changes on a branch, inviting a team member and asking for a review. If you're looking for a deeper dive into the tracking plan design capabilities of the Avo Tracking Plan, check out our <Link passHref href="/data-design/start-data-design"><a>deep dive guide on data design in Avo</a></Link>.
+
+### Step 1: Create an Avo workspace
+
+Head to our <a href="https://avo.app/onboarding" target="_blank">onboarding flow</a> and setup a new workspace.
+
+> If you already have an Avo workspace, you can skip this step.
+
+### Step 2: Import your existing tracking plan
+
+If you have an existing tracking plan that you want to import, you can do so by clicking the **Import** button in the workspace's settings.
+<br/>
+<center>
+  <img
+    src={require('../../images/audit/import-button.png')}
+    alt="Import button in Avo workspace"
+    width="450"
+  />
+</center>
+<br/>
+<br/>
+
+You can import a tracking plan from any of the following sources: Excel or Google Sheets (.csv), Mixpanel Lexicon or Amplitude Govern. Learn more about the available import formats in the <Link passHref href="/workspace/tracking-plan/importing" passHref><a>importing docs</a></Link>. If you have your tracking plan on a format that's currently not supported by the tracking plan importer, <Link passHref href="/workspace/tracking-plan/importing" passHref><a>reach out</a></Link> to us and we'll import it for you.
+
+> If you don't have an existing tracking plan, or if you just want to start fresh, you can skip this step. No worries, you can always import your existing tracking plan later.
+
+### Step 3: Create a new branch and start making changes
+
+If you are making changes to your tracking plan in Avo, we always recommend creating a branch first – no matter how small your change might be. That allows you to make suggestions in isolation from the source of truth on your main branch and get feedback from your team.
+
+Open a new branch by clicking the three line icon beside where it says "main" in the left sidebar. Then you are prompted to open an existing branch or creating a new one.
+
+<center>
+<img
+  src={require('../../images/best-practices/getting-started-to-design-data/opening-a-branch.gif')}
+  alt="Opening a branch"
+  width="450"
+/>
+</center>
+<br />
+<br />
+
+When you've created the branch, navigate to the *Events* tab in the tracking plan and create a new event by clicking the *+ Add Event* button.
+
+<center>
+<img
+  src={require('../../images/best-practices/getting-started-to-design-data/create-event.gif')}
+  alt="Creating a new event"
+  width="450"
+/>
+</center>
```

Comment:
We should probably update these gifs 😬

Reply:
Yes we should, but trying to keep the slices thin to be able to cover more ground 🙈 

I am however working on converting all gifs to mp4s in another PR since these gifs are really bad for the page performance.